specification/monitor/resource-manager/Microsoft.Insights/stable/2021-09-01/actionGroups_API.json

```diff
@@ -0,0 +1,1120 @@
+        "operationId": "ActionGroups_Update",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ActionGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "actionGroupPatch",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ActionGroupPatchBody"
+            },
+            "description": "Parameters supplied to the operation."
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "An existing action group was successfully updated.",
+            "schema": {
+              "$ref": "#/definitions/ActionGroupResource"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Patch an action group": {
+            "$ref": "./examples/patchActionGroup.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/notifications": {
+      "post": {
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "description": "Send test notifications to a set of provided receivers",
+        "operationId": "ActionGroups_PostTestNotifications",
+        "x-ms-examples": {
+          "List action groups": {
+            "$ref": "./examples/postTestNotifications.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The notification succeeded",
+            "schema": {
+              "$ref": "#/definitions/TestNotificationResponse"
+            }
+          },
+          "default": {
+            "description": "An error occurred while sending the test notifications",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/notifications/{notificationId}": {
```

Comment:
What is the purpose of this API? As there is no providers information in the API, this API will not be routed to your RP. This API sounds incorrect to me.